packages/editor/src/store/effects.js

```diff
@@ -28,32 +11,6 @@ import {
 } from './effects/reusable-blocks';
 
 export default {
-	SETUP_EDITOR( action ) {
-		const { post, edits, template } = action;
-
-		// In order to ensure maximum of a single parse during setup, edits are
-		// included as part of editor setup action. Assume edited content as
-		// canonical if provided, falling back to post.
-		let content;
-		if ( has( edits, [ 'content' ] ) ) {
-			content = edits.content;
-		} else {
-			content = post.content.raw;
-		}
-
-		let blocks = parse( content );
-
-		// Apply a template for new posts only, if exists.
-		const isNewPost = post.status === 'auto-draft';
-		if ( isNewPost && template ) {
-			blocks = synchronizeBlocksWithTemplate( blocks, template );
-		}
-
-		return [
```

Comment:
I suspect that this was the only occurrence of an array-action for which we apply the `redux-multi` middleware (i.e. upon confirmation, it may be warranted to remove as part of these changes).

Reply:
Sorry I don't follow:  The refactor changed this from an array action to [dispatch actions](https://github.com/WordPress/gutenberg/pull/14513/files#diff-a73c5f32ab7bda86e29a48b54421fc9cR70).  So the work in this pull already removes the reliance on the redux-multi middleware for `setupEditor`?  Or do you mean that we can remove the `redux-multi` dependency because of this (which makes sense).

Reply:
>Or do you mean that we can remove the redux-multi dependency because of this (which makes sense).

Yes, this.

Reply:
removed in 52af9b380d

Reply:
> I _think_ I did things correctly (just removed from package.json) but with this being a mono-repo I'm not sure :)

The one-line removal from `package-lock.json` seems correct since it's still a dependency in `block-editor` (though perhaps it shouldn't be; a point outside the scope of this pull request).

In any case, Travis would have caught it anyways if it was an issue, and it appears to be just fine.